docs/sources/features/datasources/testdata.md

```diff
@@ -9,36 +9,36 @@ weight = 20
 +++
 
 
-# Grafana TestData
+# Grafana TestData DB
 
-The purpose of this data sources is to make it easier to create fake data for any panel.
-Using `Grafana TestData` you can build your own time series and have any panel render it.
+The purpose of this data source is to make it easier to create fake data for any panel.
+Using `TestData DB` you can build your own time series and have any panel render it.
 This make is much easier to verify functionally since the data can be shared very easily.
 
 ## Enable
 
-`Grafana TestData` is not enabled by default. To enable it, first navigate to the Plugins section, found in your Grafana main menu. Click the Apps tabs in the Plugins section and select the Grafana TestData App. (Or navigate to http://your_grafana_instance/plugins/testdata/edit to go directly there). Finally click the enable button to enable.
+The `TestData DB` data source is not enabled by default. To enable it, first navigate to the *Data Source* section of the configuration menu. Click the *Add Data Source*, search for and select `TestData DB`. Finally, click *Save & Test* to enable it.
```

Comment:
Per the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md) and [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/), clickable items in the UI should be bold, not italic. Please change that. 

Also, step-by-step instructions should be in numbered step format. This will make them easier to read.

By the way, did you test whether you needed to enable the TestData DB data source? I have played with it a little, and it was always available by default.

Reply:
> clickable items in the UI should be bold, not italic

Done

> step-by-step instructions should be in numbered step format

Done

> did you test whether you needed to enable the TestData DB data source?

Yes, it appears this is required. Just tested on a brand-new installation.

Reply:
Installation on which OS?

Reply:
Grafana v6.6.1 (21bf8b71bc) on Ubuntu 19.10 (installed w/ apt)

Reply:
Okay, I figured out why I am confused. You do need to enable it...the same as every other data source. LOL.